boundary_layer_default_plugin/config/generators/list_object_generator.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Etsy Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+name: list_object_generator
```

Comment:
Is this used in this PR?  Or perhaps it is something you have used for your local testing?  If it's not used here, would you mind separating it out into a separate PR, since it has a function that is distinct from the Kubernetes functions here.

Reply:
> sed in this PR? Or perhaps it is something you have used for your local testing? If it's not

Yes it's something I use in some of our DAGs, I move it to another pull request